prisma/schema.sql

```diff
@@ -272,6 +272,7 @@ CREATE TABLE applications (
   paid_through_shopify BOOLEAN NOT NULL DEFAULT false,
   shopify_payment_status ShopifyPaymentStatus DEFAULT 'PENDING',
   shopify_confirmation_number VARCHAR(255) UNIQUE,
+  shopify_order_number VARCHAR(255),
```

Comment:
Saving `order_number` since we need both order ID and order number for these changes.

Order is currently saved in the shopifyConfirmationNumber field and gets us the url to the admin page e.g 
https://poppy-hazel.myshopify.com/admin/orders/4671298961430

Order number is for what is displayed on the frontend => "Paid through Shopify: Order #1085"

Reply:
Sounds good! Would this be unique?
```suggestion
  shopify_order_number VARCHAR(255) UNIQUE,
```